Sources/App/Websocket/SessionManager.swift

```diff
@@ -1,83 +1,65 @@
-/// Copyright (c) 2019 Razeware LLC
-///
-/// Permission is hereby granted, free of charge, to any person obtaining a copy
-/// of this software and associated documentation files (the "Software"), to deal
-/// in the Software without restriction, including without limitation the rights
-/// to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
-/// copies of the Software, and to permit persons to whom the Software is
-/// furnished to do so, subject to the following conditions:
-///
-/// The above copyright notice and this permission notice shall be included in
-/// all copies or substantial portions of the Software.
-///
-/// Notwithstanding the foregoing, you may not use, copy, modify, merge, publish,
-/// distribute, sublicense, create a derivative work, and/or sell copies of the
-/// Software in any work that is designed, intended, or marketed for pedagogical or
-/// instructional purposes related to programming, coding, application development,
-/// or information technology.  Permission for such use, copying, modification,
-/// merger, publication, distribution, sublicensing, creation of derivative works,
-/// or sale is expressly withheld.
-///
-/// THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
-/// IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
-/// FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
-/// AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
-/// LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
-/// OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
-/// THE SOFTWARE.
-
 import Vapor
 import WebSocket
 
 // MARK: For the purposes of this example, we're using a simple global collection.
 // in production scenarios, this will not be scalable beyond a single server
 // make sure to configure appropriately with a database like Redis to properly
 // scale
-final class TrackingSessionManager {
-    private(set) var sessions: LockedDictionary<TrackingSession, [WebSocket]> = [:]
+final class RoomSessionManager {
+    private(set) var sessions: LockedDictionary<RoomSession, [WebSocket]> = [:]
     
-    public static var tracking: TrackingSessionManager = {
-        return TrackingSessionManager()
+    public static var rooms: RoomSessionManager = {
+        return RoomSessionManager()
     }()
     
     // MARK: Observer Interactions
-    func add(listener: WebSocket, to session: TrackingSession) {
+    func add(listener: WebSocket, to session: RoomSession) {
+        //verify session exists
```

Comment:
nit pick

Consistency 

Correct Example: `// MARK: Poster Interactions`

Expected: `// Create session ID`

Actual: `//Create session ID`

Rule 1): Add a space
Rule 2): Capitalize the first letter